Documentation/ceph-pool-crd.md

```diff
@@ -28,6 +28,9 @@ spec:
   failureDomain: host
   replicated:
     size: 3
+    # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
+    # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
+    #targetSizeRatio: .5
```

Comment:
For the optional CR settings like this I'd prefer we not add it to the main examples, but just document it in the list of all the parameters below in the `Spec` section.